spec/integration/solo/solo_spec.rb

```diff
@@ -28,7 +28,7 @@
       file "config/solo.rb", <<~EOM
         chef_repo_path "#{@repository_dir}"
       EOM
-      result = shell_out("bundle exec chef-solo -c \"#{path_to("config/solo.rb")}\" -l debug", cwd: chef_dir)
```

Comment:
Yeah for some reason this probably can't use `--legacy-mode` so this line should probably be `bundle exec chef-solo --minimal-ohai --always-dump-stacktrace -c ...`

Reply:
(probably can't reuse the chef_solo from above)

Reply:
Updated and rebased. I'll have to close and reopen the PR to trigger build?